src/api/error.ts

```diff
@@ -0,0 +1,7 @@
+import { Response } from 'node-fetch';
+
+export class HttpResponseError extends Error {
```

Comment:
Looking at OneSpan's error schema, we could improve this error class later on to set `code`, `messageKey` and `messages` as the class' public property

<img width="912" alt="Screen Shot 2021-12-17 at 10 54 52 AM" src="https://user-images.githubusercontent.com/16750783/146571994-f089934f-de74-40c6-ac94-55f29ec695b6.png">

Reply:
You'd put that in OneSpan error specific class, right?

Reply:
Yeah, or just rename this class to something like `OneSpanError`, unless we also make calls to anything other than OneSpan's endpoints